src/jdk.compiler/share/classes/com/sun/tools/javac/comp/ReflectMethods.java

```diff
@@ -307,6 +309,7 @@ public void visitReference(JCMemberReference tree) {
         }
     }
 
+    // @@@: Only used for quoted lambda, not quotable ones. Remove?
```

Comment:
Might be worth inlining the method (see also comment related to `// @@@ just make translation happy`)